keyboards/ergodash/rev1/keymaps/rmarinbe/keymap.c

```diff
@@ -0,0 +1,140 @@
+#include QMK_KEYBOARD_H
+extern keymap_config_t keymap_config;
```

Comment:
This is not needed, as it's set elsewhere, and included by default. 
```suggestion
```

Reply:
done